### PR TITLE
MESG Orchestrator + Privacy policy (WIP)

### PR DESCRIPTION
**Pull request by EmmanuelDrouin**

### CODE REVIEW

**Comment by antho1404** [comment]
Deploy preview for *mesg* ready!

Built with commit c67145920d13cffa2ce99470422f8681c12bbfb5

https://deploy-preview-161--mesg.netlify.com

**Comment by antho1404** [comment]
@EmmanuelDrouin Is it still WIP? what's left to do?
Maybe you can put a todo list on the description

**Comment by EmmanuelDrouin** [comment]
- [ ] Add transition at the opening of the popup

- [ ] Change newsletter section in the footer

**Comment by EmmanuelDrouin** [comment]
@antho1404 can you tell me if the transition (duration and effect) is ok for you for the newsletter popup?

**Comment by antho1404** [comment]
that's great for the animation. I have some feedback on the text, I'm not really convinced by the text but I will discuss with @willcreatedit for that

**Comment by antho1404** [comment]
please don't forget to also rename the page in orchestrator instead of worfklow ;)

**Comment by EmmanuelDrouin** [comment]
> please don't forget to also rename the page in orchestrator instead of worfklow ;)

Yes will do that when i will integrate the final content ;)

**Comment by EmmanuelDrouin** [comment]
@antho1404 ready for the final review before merge

**Comment by EmmanuelDrouin** [comment]
@antho1404 the content has been updated and the page renamed to Orchestrator instead of Workflows. Ready to merge.

**Comment by antho1404** [comment]
@willcreatedit I have a few feedback on the text.

> Create any feature with a configuration file 

should be "Create any feature with a process file"

> Build apps using any feature

I understand it like build app with the features that provide MESG which is true but i'm not sure that the message we want to send

> Currently, services are managed by creating a process through a config file

We don't need to use config file. We could have process file to describe this "config"

**Comment by willcreatedit** [comment]
> > Build apps using any feature
> 
> I understand it like build app with the features that provide MESG which is true but i'm not sure that the message we want to send

@antho1404 How about something like "Orchestrate services to create application features. After all, what is an application but a collection of features?" - if that isn't the right message either, then what do "Feature-based applications" mean to you?

**Comment by antho1404** [comment]
I already prefer that, it looks like we are building features and the original text is we are building applications with some specific feature that helps us build it.

What about "Orchestrate services to create features for your application"

**Comment by willcreatedit** [comment]
@antho1404 - I like your version - Changes made 👍

File: pages/mesg-orchestrator.vue [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    </Header>
+
+    <transition name="fade">
+      <div id="popup-newsletter" v-if="popup">
+        <div class="background" @click="popup = false" />
+        <div class="content">
+          <NewsletterPopup
+            class="newsletter"
+            title="Keep in touch"
+            description="Stay connected with us and receive a notification on September 12th for the launch of MESG Orchestrator."
```
Comment (antho1404) [inline pages/mesg-orchestrator.vue]:
```suggestion
            description="Stay connected with us and receive a notification on September 19th for the launch of MESG Orchestrator."
```

File: pages/mesg-orchestrator.vue [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+              />
+            </div>
+          </div>
+        </Container>
+      </div>
+    </section>
+
+    <CallToAction
+      mb3
+      title="Launching September 12th"
```
Comment (antho1404) [inline pages/mesg-orchestrator.vue]:
```suggestion
      title="Launching September 19th"
```

**Pull request merged**